external/proto/pop.proto

```diff
@@ -165,6 +165,7 @@ message GetFinalStatementsReply {
 message StoreInstanceID {
   required bytes partyid = 1;
   required bytes instanceid = 2;
+  required bytes darcid = 3;
```

Comment:
Where did these come from? They seem unrelated to this change? (also in GetInstanceIDReply below)

Reply:
It's the result of re-running `proto.sh`. I think one of the earlier PRs changed a `proto.go` file but didn't re-run `proto.sh`.